docs/scos/dev/front-end-development/202212.0/oryx/oryx-versioning.md

```diff
@@ -72,3 +72,24 @@ Minor releases are fully backward compatible and do not require any developer as
 ### Deprecation practices
 
 If a newer and better alternative to a feature is introduced in a minor release, the old one is deprecated. Deprecated features remain functional until the next major release, in which they are removed completely.
+
+## Feature flag versioning
+
+Feature flag versioning in Oryx enables you to roll out new functionalities, like changes in page structure or additional component options, while ensuring backward compatibility. Feature flag versioning is directly aligned with major and minor releases, allowing for a balanced mix of innovation and stability.
+
+### ORYX_FEATURE_VERSION
+
+The `ORYX_FEATURE_VERSION` environment variable is used to opt into new functionalities or to lock and maintain compatibility with existing setups. It optimizes your build by leveraging Dead Code Elimination. Any code unrelated to the specified version isn't included in the final bundle, making your application leaner.
+
+The following example shows how you can enable a specific version:
+
+```
+ORYX_FEATURE_VERSION=1.1
+```
+
+You can also use the `featureVersion` utility in your code or third-party libraries to conditionally enable features.
```

Comment:
Do I understand correctly that ORYX_FEATURE_VERSION is only for code, while featureVersion is boith for code and third-party libraries?